<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>780</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QPushButton {
        background-color: #D11A0B;
        color: white;
        border-radius: 20px;
        padding: 0px;
        border: 1px solid #D11A0B;
        font-size: 12px;
    }
    QPushButton:hover {
        background-color: #FFFFFF;
        color: #D11A0B;
    }
    QPushButton:pressed {
        background-color: #FFFFFF;
        color: #D11A0B;
    }
    QTableWidget {
        background-color: #FFFFFF;
        border: 1px solid #dcdcdc;
        gridline-color: #dcdcdc;
    }
    QTableWidget QHeaderView::section {
        background-color: #D11A0B;
        color: #FFFFFF;
        padding: 5px;
        border: 1px solid #D11A0B;
    }
   </string>
  </property>
  <widget class="QPushButton" name="ButtonAccueil">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Accueil</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonGestion">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Gestion des accès</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonPerimetres">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Périmètres</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ButtonZones">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>90</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Identification zones</string>
   </property>
  </widget>
  <widget class="QListView" name="listView_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>200</y>
     <width>551</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>30</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Périmètres</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetAcces">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>201</y>
     <width>531</width>
     <height>491</height>
    </rect>
   </property>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column>
    <property name="text">
     <string>Nom</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Prénom</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Mail</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Acces périmètre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Bâtiment</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background-color: gray;
	color: #ffffff; /* Couleur du texte (blanc dans cet exemple) */
    border: none;
}
</string>
   </property>
   <property name="text">
    <string>Listes des accès</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
